packages/communication/src/getChainConfig.ts

```diff
@@ -19,6 +19,22 @@ const presets = {
     ticker: 'MATIC',
     tickerName: 'Matic',
   },
+  sepolia: {
```

Comment:
```suggestion
  baseSepolia: {
```
Just `sepolia` is the name for Ethereum testnet